### PR TITLE
Simplified Chinese translation

### PR DESCRIPTION
**Pull request by Cruel**
Thanks to pdapanda on GBATemp.

### CODE REVIEW

File: res/romfs/lang/zh.lang [hunk 1]
```diff
@@ -0,0 +1,395 @@
+# src/States/SyncState.cpp
+##########################
+Waiting for internet connection... %.0fs
+等待网络连接... %.0fs
+
+Fetching news for %s ...
+正在获取消息 \n%s ...
```
Comment (arition) [inline res/romfs/lang/zh.lang]: 正在获取%s的更新内容...

File: res/romfs/lang/zh.lang [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+
+Loading game list...
+加载列表...
```
Comment (arition) [inline res/romfs/lang/zh.lang]: 加载游戏列表...

File: res/romfs/lang/zh.lang [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+
+Looking for freeShop update...
+查询freeshop版本升级...
```
Comment (arition) [inline res/romfs/lang/zh.lang]: 检查freeShop是否有更新...

File: res/romfs/lang/zh.lang [hunk 4]
```diff
@@ -0,0 +1,395 @@
...
+
+Installing freeShop %s ...
+安装 freeShop %s ...
+
+Failed to install update: 0x%08lX
+安装升级文件失败: 0x%08lX
```
Comment (arition) [inline res/romfs/lang/zh.lang]: 无法安装更新: 0x%08lX

File: res/romfs/lang/zh.lang [hunk 5]
```diff
@@ -0,0 +1,395 @@
...
+Queued install: %s
+等待安装: %s
+
+Already installed: %s
+已经安装: %s
+
+# src/States/DialogState.cpp
+############################
+ Ok
+ 好的
```
Comment (arition) [inline res/romfs/lang/zh.lang]:  确认

Comment (arition) [inline res/romfs/lang/zh.lang]: just 好的->确认. I have no idea why github give me a .

File: res/romfs/lang/zh.lang [hunk 6]
```diff
@@ -0,0 +1,395 @@
...
+ Retry
+ 重试
+
+# src/AppInfo.cpp
+#################
+No Screenshots
+没有截图
+
+No game selected
+没有游戏被选中
```
Comment (arition) [inline res/romfs/lang/zh.lang]: 没有选择游戏

File: res/romfs/lang/zh.lang [hunk 7]
```diff
@@ -0,0 +1,395 @@
...
+
+Demo
+试玩版
+
+You are deleting the demo for\nthis title:\n\n%s
+你正在删除\n这个游戏的试玩版:\n\n%s
+
+Deleted demo: %s
+被删除的试玩版: %s
```
Comment (arition) [inline res/romfs/lang/zh.lang]: 已删除试玩版: %s

File: res/romfs/lang/zh.lang (hunk 1 above)
Comment (arition) [inline res/romfs/lang/zh.lang]: 消息 refers to message, not news.

File: res/romfs/lang/zh.lang (hunk 2 above)
Comment (arition) [inline res/romfs/lang/zh.lang]: missing ```game```

File: res/romfs/lang/zh.lang (hunk 3 above)
Comment (arition) [inline res/romfs/lang/zh.lang]: more formal

File: res/romfs/lang/zh.lang (hunk 4 above)
Comment (arition) [inline res/romfs/lang/zh.lang]: more formal

File: res/romfs/lang/zh.lang (hunk 5 above)
Comment (arition) [inline res/romfs/lang/zh.lang]: more formal

File: res/romfs/lang/zh.lang (hunk 6 above)
Comment (arition) [inline res/romfs/lang/zh.lang]: More formal

File: res/romfs/lang/zh.lang (hunk 7 above)
Comment (arition) [inline res/romfs/lang/zh.lang]: More formal

File: res/romfs/lang/zh.lang [hunk 8]
```diff
@@ -0,0 +1,395 @@
...
+
+Queued demo: %s
+等待安装的试玩版: %s
```
Comment (arition) [inline res/romfs/lang/zh.lang]:
```
等待安装试玩版: %s
```

Comment (arition) [inline res/romfs/lang/zh.lang]: More formal

File: res/romfs/lang/zh.lang [hunk 9]
```diff
@@ -0,0 +1,395 @@
...
+正在安装 TMD...
+
+Importing content...
+导入数据....
+
+Installing %d/%d... %.1f%% (%.0f KB/s) %dm %02ds
+正在安装 %d/%d... %.1f%% (%.0f KB/s) %dm %02ds
+
+Suspended
+中断
```
Comment (arition) [inline res/romfs/lang/zh.lang]:
```
暂停
```

Comment (arition) [inline res/romfs/lang/zh.lang]: 中断 is more like stop. 暂停 is better.

File: res/romfs/lang/zh.lang [hunk 10]
```diff
@@ -0,0 +1,395 @@
...
+升级文件
+
+DLC
+DLC
+
+You are deleting updates for\nthis title:\n\n%s
+你正在删除\n这个游戏的升级文件:\n\n%s
+
+Deleted update: %s
+已删除的升级文件: %s
```
Comment (arition) [inline res/romfs/lang/zh.lang]:
```
已删除升级文件: %s
```

Comment (arition) [inline res/romfs/lang/zh.lang]: More formal

File: res/romfs/lang/zh.lang [hunk 11]
```diff
@@ -0,0 +1,395 @@
...
+
+Queued update: %s
+等待安装的升级文件: %s
+
+You are deleting DLC for\nthis title:\n\n%s
+你正在删除\n这个游戏的DLC:\n\n%s
+
+Deleted DLC: %s
+已删除的DLC: %s
```
Comment (arition) [inline res/romfs/lang/zh.lang]:
```
已删除DLC: %s
```

Comment (arition) [inline res/romfs/lang/zh.lang]: More formal

File: res/romfs/lang/zh.lang [hunk 12]
```diff
@@ -0,0 +1,395 @@
...
+需要9.6以上固件, seed错误: %016llX
+
+Failed to start: 0x%08lX
+启动失败: 0x%08lX
+
+Failed to resume: 0x%08lX
+继续安装失败: 0x%08lX
+
+Failed to suspend: 0x%08lX
+中断失败: 0x%08lX
```
Comment (arition) [inline res/romfs/lang/zh.lang]:
```
暂停失败: 0x%08lX
```

Comment (arition) [inline res/romfs/lang/zh.lang]: 中断 is more like stop. 暂停 is better.

File: res/romfs/lang/zh.lang [hunk 13]
```diff
@@ -0,0 +1,395 @@
...
+
+Failed to commit title: 0x%08lX
+提交游戏名失败: 0x%08lX
```
Comment (arition) [inline res/romfs/lang/zh.lang]:
```
提交title失败: 0x%08lX
```

Comment (arition) [inline res/romfs/lang/zh.lang]: There is no formal translation for ```title``` in 3DS hacking in Chinese. Therefore, it should be stay in English.

File: res/romfs/lang/zh.lang [hunk 14]
```diff
@@ -0,0 +1,395 @@
...
+Failed to install TMD: 0x%08lX
+安装TMD失败: 0x%08lX
+
+Failed to install content: 0x%08lX
+安装数据失败: 0x%08lX
+
+# src/TitleKeys.cpp
+###################
+Failed.\n\nHTTP Status: %d
+失败.\n\nHTTP 状况: %d
```
Comment (arition) [inline res/romfs/lang/zh.lang]:
```
失败.\n\nHTTP状态码: %d
```

Comment (arition) [inline res/romfs/lang/zh.lang]: More accurate translation.
[zh wikipedia](https://zh.wikipedia.org/wiki/HTTP%E7%8A%B6%E6%80%81%E7%A0%81)
[en wikipedia](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes)

File: res/romfs/lang/zh.lang [hunk 15]
```diff
@@ -0,0 +1,395 @@
...
+  类  型  
+
+Language
+  语  言  
+
+Platform
+  平  台  
+
+Sort
+  整  理
```
Comment (arition) [inline res/romfs/lang/zh.lang]:
```
  排  序  
```

Comment (arition) [inline res/romfs/lang/zh.lang]: More accurate translation.

**Comment by arition** [comment]
Final lang file

[zh.lang.txt](https://github.com/Cruel/freeShop/files/485740/zh.lang.txt)

**Comment by Cruel** [comment]
Thanks!

**Pull request merged**